pytorch_lightning/trainer/trainer.py

```diff
@@ -80,6 +81,8 @@
     'please use torch.distributed.ReduceOp instead'
 )
 
+NEW_LOOP = True
```

Comment:
we could use an env variable to get this?

Reply:
this is just for us to trigger the tests properly. Will be removed once final (will always use new Loop)

Reply:
want me to remove in this PR already in this PR or when we remove the old loop class in the next PR?

Reply:
@awaelchli I think still keeping it is good since it also routes to the correct validation loop